sqs/provider.go

```diff
@@ -388,6 +463,23 @@ func (s *Provider) getStack(ctx context.Context, stackName string) (*cloudformat
 	return state, nil
 }
 
+// tryDestroyStack removes the cloudformation stack by name. It does not use
+// the request's context to avoid the siutation where a timeout has been
+// reached and the stack needs to be cleaned up.
+// It is intended to be run as a one off goroutine at a point when no error can be returned
+// so can only log any problems.
+func (s *Provider) tryDestroyStack(stackName string) {
+	deleteCtx := context.Background()
```

Comment:
this line seems like a noop?

Reply:
it gets wrapped in the next line ... could be written as a single line, guess I have a habbit for wrapping contexts line by line like this

Reply:
oh yeah I somehow didn't see that. move along